torch/_dynamo/symbolic_convert.py

```diff
@@ -386,6 +386,9 @@ def wrapper(self: "InstructionTranslatorBase", inst: Instruction):
             state = self.copy_graphstate()
             reason = None
             try:
+                TracingContext.set_current_loc(
+                    self.f_code.co_filename, self.lineno, self.f_code.co_name
+                )
```

Comment:
This is necessary because I've removed the `self.frame_summary()` manual addition in the except handling. This handling is better as it ensures regular exceptions also get the current location setup correctly as well.